addons/binding/org.openhab.binding.irtrans/OSGI-INF/IRtransHandlerFactory.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2014 openHAB UG (haftungsbeschraenkt) and others.
```

Comment:
> `mvn licence:format`

Have a try with `mvn license:format` instead :-)
Of if it was only a typo in your comment, maybe try removing the empty line 3.